src/components/RadioGroup/RadioGroup.stories.tsx

```diff
@@ -75,6 +76,25 @@ export const Controls: Story = {
     },
     jest: ["RadioGroup.test.tsx"],
   },
+  play: async ({ canvasElement }) => {
+    const screen = within(canvasElement);
+
+    expect(screen.getByRole("radiogroup")).toBeInTheDocument();
+
+    expect(screen.getByLabelText("Radio 4")).toBeChecked();
+
+    await userEvent.click(screen.getByLabelText("Radio 3"));
+    await userEvent.click(screen.getByLabelText("Radio 2"));
+    await userEvent.click(screen.getByLabelText("Radio 5"));
+    expect(screen.getByLabelText("Radio 5")).toBeChecked();
+
+    await userEvent.keyboard("{arrowdown}");
+    await userEvent.keyboard("{arrowleft}");
+    await userEvent.keyboard("{arrowup}");
+    await userEvent.keyboard("{arrowright}");
+    await userEvent.keyboard("{arrowdown}");
+    expect(screen.getByLabelText("Radio 2")).toBeChecked();
```

Comment:
Odd, I _now_ see this in the UI. 
![Screenshot 2024-06-05 at 3 04 28 PM](https://github.com/NYPL/nypl-design-system/assets/1280564/1fe14672-c86a-4be8-b4cb-e447c4bd89fd)